text/0118-search-api.md

```diff
@@ -460,14 +460,182 @@ Search queries using `_geoPoint` returns a `_geoDistance` field containing the d
 - Type: Object
 - Required: False
 
-`_formatted` returns highlighted and cropped attributes specified in `attributesToHighlight` and/or `attributesToCrop` of a search result.
+`_formatted` is an object returned in the search response, only if at least one of the following paramaters has been set in the search query:
+- `attributesToHighlight`
+- `attributesToCrop`
+
+If `attributesToHighlight` and `attributesToCrop` are not set, `_formatted` is not returned.
+
+This `_formatted` object will be present in the each returned document in the `hits` field.
+
+Example:
+
+```json
+{
+    "q": "",
```

Comment:
```suggestion
```

This is unnecessary and might give the impression that you **need** to do a query to get a formatted field.